src/styles/main.scss

```
@@ -10,5 +10,7 @@ body {
   box-sizing: border-box;
 }
 
+@import "./utils/variables";
```

If you will add bottom dash to all folders, it would be in same code style
```suggestion
@import "./_utils/variables";
```

---

good point